runtime-light/state/instance-state.cpp

```diff
@@ -83,8 +84,9 @@ task_t<void> InstanceState::run_instance_prologue() noexcept {
     using namespace PhpServerSuperGlobalIndices;
     superglobals.v$_SERVER.set_value(string{REQUEST_TIME.data(), REQUEST_TIME.size()}, static_cast<int64_t>(time_mcs));
     superglobals.v$_SERVER.set_value(string{REQUEST_TIME_FLOAT.data(), REQUEST_TIME_FLOAT.size()}, static_cast<double>(time_mcs));
+    superglobals.v$d$PHP_SAPI = system_functions_impl_::php_sapi_name(image_kind_);
```

Comment:
let's better implement this as a template function (`template <ImageKind kind>`) here. Thus we remove the number of includes and therefore the number of inner dependencies